src/Main.elm

```diff
@@ -3,10 +3,15 @@ module Main exposing (main)
 import Browser
 import Browser.Dom
 import Browser.Navigation
-import Html.Styled exposing (Html, h1, text, toUnstyled)
+import Html.Styled as Html exposing (..)
```

Comment:
We'll eventually want to agree how we do this. For Html I prefer the explicit as

`import Html.Styled exposing (Html, h1, text, toUnstyled)`

If we don't stick to one standard, we'll end up spending time going back and forth.